config/lava/chromeos/chromeos-tast-template.jinja2

```diff
@@ -0,0 +1,53 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block metadata %}
+{{ super() }}
+{% endblock %}
+{% block main %}
+{{ super() }}
+{% endblock %}
+{% block actions %}
+actions:
+  - deploy:
+      timeout:
+        minutes: 30
+      to: tftp
+      kernel:
+        url: {{ kernel_url }}
+      os: oe
+
+  - boot:
+      timeout:
+        minutes: 30
+      method: depthcharge
+      commands: emmc
+      prompts:
+        - 'sof-audio-pci'  # ToDo: make a parameter
+
+  - test:
+      timeout:
+        minutes: 45
+      docker:
+        image: kernelci/chromeos-tast  # ToDo: make a parameter
+        volumes:
+          - /tmp/octopus:/home/cros-tast/octopus  # ToDo: make a parameter
```

Comment:
I don't think this is actually used, as the tarball is downloaded over HTTP from the host.